Scripts/DCS-BIOS/test/compile/Log.lua

```diff
@@ -0,0 +1,10 @@
+module("Log", package.seeall)
+
+--- @class Log
+Log = {}
```

Comment:
we should probably avoid declaring this as global, otherwise tests could pass when we forgot to require something
```suggestion
local Log = {}
```

Reply:
ouch